src/shims/rand.rs

```diff
@@ -0,0 +1,19 @@
+use crate::*;
+
+impl<'mir, 'tcx: 'mir> EvalContextExt<'mir, 'tcx> for crate::MiriInterpCx<'mir, 'tcx> {}
+pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
+    fn getentropy(
```

Comment:
Please move this to `macos/foreign_items` and call it from `dlsym`. It is macos-specific and should live in the macos module.

Reply:
Ah I can do that. I was following the pattern of `mach_timebase_info` but I guess there's not enough in this file for it to be worth existing.

Reply:
Ah yeah for time we have all the shims for all OSes in one file. Those are old shims, we've been adjusting and organizing them differently more recently. (Also, following that pattern would mean putting all getrandom shims for all OSes into `shims/rand.v`.)

Reply:
Alrighty, its been moved to the macOS-specific module.